### PR TITLE
Create container to save markdown files

### PR DESCRIPTION
**Pull request by jwmcconnell**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import DocumentContainer from '../containers/DocumentContainer';
+import DocumentList from './controls/DocumentList';
 
 export default function App() {
   return (
     <>
       <h1>Markdown Editor</h1>
+      <DocumentList documents={[]} />
```
Comment (Zilula) [inline src/components/App.js]: this shouldnt take an empty array. it should probably take the prop thats being passed from redux

Comment (jwmcconnell) [reply src/components/App.js]: We figured that was part of the next ticket "Create a container that uses Redux to display a list of markdown files."

Comment (Zilula) [reply src/components/App.js]: dont render until this has actual documents

Comment (Zilula) [reply src/components/App.js]: the hope is that dev can always be merged to master for hot fixes

Comment (jwmcconnell) [reply src/components/App.js]: Added that in

**Review by Zilula — APPROVED** [review]

**Review by dannyhogan — APPROVED** [review]
Looks good!

**Pull request merged**